woocommerce-services.php

```diff
@@ -514,6 +500,33 @@ public function load_textdomain() {
 			load_plugin_textdomain( 'woocommerce-services', false, dirname( plugin_basename( __FILE__ ) ) . '/i18n/languages' );
 		}
 
+		public function jetpack_on_plugins_loaded() {
+			if ( false === WC_Connect_Jetpack::is_jetpack_version_supported() ) {
```

Comment:
I'm not sure that this is needed at all. Even if an old version of Jetpack is installed, the Autoloader will smartly load the newest version of `jetpack-connection` (the one bundled with WCS&T).

Reply:
Oh, interesting, thanks! I saw the same check in WCPay and when I tested this alongside the Jetpack plugin, I noticed that the JP plugin's classes were loaded. I assumed that the plugin's classes were taking precedence.
But your comment aligns with https://github.com/Automattic/jetpack/tree/master/projects/packages/autoloader - so that's great to know! :) 

> This is a custom autoloader generator that uses a classmap to always load the latest version of a class.

Reply:
> I saw the same check in WCPay

The autoloader hasn't always worked flawlessly :)

I remember that, if the Jetpack version is REALLY old, it could take precedence over the Composer dependency. That mattered last year, but at this point if the merchant is using a 2+ year old version of Jetpack I'm ok with it crashing.

Reply:
Here's the explanation, it's your call if you think avoiding a crash on JP 8.1 is worth it: https://github.com/Automattic/woocommerce-payments/blob/16c5a7c9b713fc937c46955ca33afcc27c74fda2/woocommerce-payments.php#L102-L104